apps/meteor/client/views/room/composer/messageBox/MessageBox.tsx

```diff
@@ -117,7 +122,60 @@ const MessageBox = ({
 	const composerPlaceholder = useMessageBoxPlaceholder(t('Message'), room);
 
 	const [typing, setTyping] = useReducer(reducer, false);
+	const [uploadfiles, setUploadfiles] = useState<File[]>([]);
+	const dispatchToastMessage = useToastMessageDispatch();
+	const maxFileSize = useSetting('FileUpload_MaxFileSize') as number;
+	function handlefileUpload(fileslist: File[], resetFileInput?: () => void) {
+		setUploadfiles((prevFiles) => [...prevFiles, ...fileslist]);
 
+		resetFileInput?.();
+	}
+
+	const handleremove = (index: number) => {
+		if (!uploadfiles) {
```

Comment:
This check is not necessay since uploadfiles will always be an array

Reply:
yes sir i have removed it